questions/unit-testing.md

```diff
@@ -0,0 +1,21 @@
+### What is Unit Testing ?
+
+### Answer
+
+In layman terms, unit Testing is a level of software testing where individual units or components of a software are tested independently.
+
+Generally, we write code tests using frameworks like Mocha and Chai. We validate the module by giving it all types of inputs into it, and expect the output we are looking for.
+
+**Benefits**:
```

Comment:
This section should be in Good to hear

Reply:
The Benefits ?

Reply:
Yup. You wouldn't mention them as part of the answer, but it would be good to hear and even good to research on your own.

Reply:
Done ! @skatcat31